virttest/qemu_monitor.py

```diff
@@ -263,7 +263,7 @@ def __init__(self, vm, name, filename, suppress_exceptions=False):
         :raise MonitorConnectError: Raised if the connection fails
         """
         self.vm = VM(vm.name)
-        self._enable_blockdev = vm.check_capability(Flags.BLOCKDEV)
+        self._enable_blockdev = False if type(vm) == VM else vm.check_capability(Flags.BLOCKDEV)
```

Comment:
@pevogam hi, I'd suggest making it to be handled by the dummy VM class, like:
```patch
diff --git a/virttest/qemu_monitor.py b/virttest/qemu_monitor.py
index dd0fe9de..7c00b10d 100644
--- a/virttest/qemu_monitor.py
+++ b/virttest/qemu_monitor.py
@@ -241,6 +241,12 @@ class VM(object):
     def __init__(self, name):
         self.name = name
 
+    def check_capability(self, flag):
+        return False
+
+    def get_pid(self):
+        return None
+
 
 class Monitor(object):
 
```

Reply:
Very good point, updated.